### DISCUSSION TITLE
Is there a way to enable VCPKG_DISABLE_COMPILER_TRACKING and --x-abi-tools-use-exact-versions globally

### DISCUSSION THREAD

**Discussion by xiaozhuai** [microsoft/vcpkg#33187] [Q&A] [answered]
See https://github.com/microsoft/vcpkg/issues/26346#issuecomment-1319244766
I want to reduce cache missing.
Is there a way to enable `VCPKG_DISABLE_COMPILER_TRACKING` and `--x-abi-tools-use-exact-versions` globally? (Through environment variable?)
I didn't like creating a triplet for adding VCPKG_DISABLE_COMPILER_TRACKING and passing --x-abi-tools-use-exact-versions during installation.

**Comment by xiaozhuai** [comment]
@Osyotr Excuse me, any suggestions about this?

**Reply by Osyotr** [reply, marked as answer]
> `VCPKG_DISABLE_COMPILER_TRACKING`

I don't think so (this variable is hardcoded in vcpkg-tool), but you can create your own triplet with this variable and make it default https://learn.microsoft.com/en-us/vcpkg/users/config-environment#vcpkg_default_triplet 

> `--x-abi-tools-use-exact-versions`

I don't think there's a way either, although IMHO this option should be enabled by default.

The demand is reasonable, feel free to open a PR in vcpkg-tool repo.

**Reply by xiaozhuai** [reply]
@Osyotr Thanks!
